src/main/kotlin/uk/gov/dwp/dataworks/kafkatohbase/reconciliation/configuration/MetadataStoreConfiguration.kt

```diff
@@ -45,7 +45,7 @@ data class MetadataStoreConfiguration(
             }
         }
 
-        HBaseConfiguration.logger.info("Metastore Configuration loaded", "metastore_properties" to properties.toString())
+        logger.info("Metastore Configuration loaded", "metastore_properties" to properties.toString())
```

Comment:
Can we change it from "Metastore" to "Metadata Store" just to keep inline with naming elsewhere?

Reply:
Done.